### ISSUE TITLE
Implement R&R Type 1 analysis

### ISSUE THREAD

**Issue by manniealfaro** [emilopezcano/SixSigma#6]
Implement a new function for R&R Type 1 analysis, also cover the possibility to use only one spec limit.

**Comment by k-tatgenhorst** [comment]
I don't believe my statistical skills in R are up to this challenge yet, but we currently can compare the variation between parts and between appraisers. Does this fall short of what a type 1 analysis provides? I thought we were covering both the measuring system and the appraiser - is a type 1 just omitting the appraiser or is there more to it?

**Comment by emilopezcano** [comment]
Hi Karl,
The type 1 gage study is somehow different (and simpler). It is needed a calibrated "real" reference, and then assess the measurement error (not the sources of variation). 

Check this post: https://www.engineering.com/story/gage-studies-and-gage-rr

And this example in Minitab: https://support.minitab.com/minitab/21/help-and-how-to/quality-and-process-improvement/measurement-system-analysis/how-to/gage-study/type-1-gage-study/before-you-start/example/
